modin/experimental/core/execution/native/implementations/hdk_on_native/calcite_builder.py

```diff
@@ -377,11 +378,39 @@ def gen_reduce_expr(self):
                 self._dtype,
             )
 
+    class QuantileAggregate(CompoundAggregateWithColArg):
+        """
+        A QUANTILE aggregate generator.
+
+        Parameters
+        ----------
+        builder : CalciteBuilder
+            A builder to use for translation.
+        arg : List of BaseExpr
+            An aggregated values.
```

Comment:
let's add some description about parameters
```suggestion
        arg : List of BaseExpr
            A list of 3 values. First - a column to compute the quantiles for, second - ..., third - ...
```